Woof/WoofTests/Model/Preferences/PreferencesHandlerSaveTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+
+final class PreferencesHandlerSaveTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        PreferencesHandlerTestHelper.cleanStorage()
+    }
+
+    override func tearDown() {
+        PreferencesHandlerTestHelper.cleanStorage()
+        super.tearDown()
+    }
+
+    func testSaveSelectedRoleExistInAPI() {
+        _ = PreferencesHandler.saveSelectedRole(.sitter)
+    }
+
+    func testSaveSelectedRoleReturnsTrueWhenUsingWithAllCases() {
+        Role.allCases.forEach { role in
+            XCTAssertTrue(PreferencesHandler.saveSelectedRole(role))
+        }
+    }
```

Comment:
Also couldn't clearly understand what you want to check. I need your clarification.

Reply:
I check if the method successfully saves each possible role case

Reply:
I changed the test